sevntu-checks/.ci/wercker.sh

```diff
@@ -21,149 +21,164 @@ function checkout_from {
+  groovy ./diff.groovy --listOfProjects projects-for-wercker.properties \
+      --patchConfig checks-sevntu-error.xml --checkstyleVersion ${CS_POM_VERSION} \
+      --sevntuVersion ${SEVNTU_POM_VERSION} --allowExcludes \
+      --mode single --patchBranch "$BRANCH"
   cd ../../
   rm -rf contribution
   ;;
 
 no-exception-guava)
+  BRANCH=$(git rev-parse --abbrev-ref HEAD)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' \
                      -Dexec.args='${checkstyle.eclipse-cs.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   SEVNTU_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
   echo SEVNTU_version: ${SEVNTU_POM_VERSION}
-  checkout_from https://github.com/checkstyle/contribution.git
+  checkout_from "-b issue-529 https://github.com/nmancus1/contribution.git"
   cd .ci-temp/contribution/checkstyle-tester
   sed -i'' 's/^guava/#guava/' projects-for-wercker.properties
   sed -i'' 's/#guava/guava/' projects-for-wercker.properties
-  groovy ./launch.groovy --listOfProjects projects-for-wercker.properties \
-      --config checks-sevntu-error.xml --checkstyleVersion ${CS_POM_VERSION} \
-      --sevntuVersion ${SEVNTU_POM_VERSION}
+  groovy ./diff.groovy --listOfProjects projects-for-wercker.properties \
+      --patchConfig checks-sevntu-error.xml --checkstyleVersion ${CS_POM_VERSION} \
+      --sevntuVersion ${SEVNTU_POM_VERSION}  --allowExcludes \
+      --mode single --patchBranch "$BRANCH"
   cd ../../
   rm -rf contribution
   ;;
 
 no-exception-hibernate-orm)
+  BRANCH=$(git rev-parse --abbrev-ref HEAD)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' \
                      -Dexec.args='${checkstyle.eclipse-cs.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   SEVNTU_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
   echo SEVNTU_version: ${SEVNTU_POM_VERSION}
-  checkout_from https://github.com/checkstyle/contribution.git
+  checkout_from "-b issue-529 https://github.com/nmancus1/contribution.git"
   cd .ci-temp/contribution/checkstyle-tester
   sed -i.'' 's/^guava/#guava/' projects-to-test-on.properties
   sed -i.'' 's/#hibernate-orm/hibernate-orm/' projects-to-test-on.properties
-  groovy ./launch.groovy --listOfProjects projects-for-wercker.properties \
-      --config checks-sevntu-error.xml --checkstyleVersion ${CS_POM_VERSION} \
-      --sevntuVersion ${SEVNTU_POM_VERSION}
+  groovy ./diff.groovy --listOfProjects projects-for-wercker.properties \
+      --patchConfig checks-sevntu-error.xml --checkstyleVersion ${CS_POM_VERSION} \
+      --sevntuVersion ${SEVNTU_POM_VERSION}   --allowExcludes \
```

Comment:
Here is usage of `--checkstyleVersion` and `--sevntuVersion`.

Reply:
lets use maven extra options of diff.groovy